src/iop/overexposed.c

```diff
@@ -148,7 +148,7 @@ static void _transform_image_colorspace(dt_iop_module_t *self, const float *cons
       = dt_ioppr_add_profile_info_to_list(self->dev, darktable.color_profiles->display_type,
                                           darktable.color_profiles->display_filename, INTENT_PERCEPTUAL);
   const dt_iop_order_iccprofile_info_t *const profile_info_to
-      = dt_ioppr_add_profile_info_to_list(self->dev, histogram_type, histogram_filename, INTENT_PERCEPTUAL);
+      = dt_ioppr_add_profile_info_to_list(self->dev, histogram_type, histogram_filename, INTENT_RELATIVE_COLORIMETRIC);
```

Comment:
Why changing the intent here?

Reply:
Perceptual is black magic and poorly suited for gamut checking. For media profiles, it uses a vendor-dependent LUT to do the gamut mapping, along with some S curve to perform the black point adaptation. But for standard RGB spaces (without that LUT defined), what they do is undefined.

Relative colorimetric is more predictable, since it desaturates out-of-gamut colors at same luminance, until they reach the target gamut. Since we are here doing diagnostic over clipping, relative colorimetric is just more reliable.